cranelift/isle/isle/src/parser.rs

```diff
@@ -202,7 +202,7 @@ impl<'a> Parser<'a> {
     fn parse_const(&mut self) -> Result<Ident> {
         let pos = self.pos();
         let ident = self.parse_ident()?;
-        if ident.0.starts_with("$") {
+        if ident.0.starts_with('$') {
             let s = &ident.0[1..];
```

Comment:
As long as we're changing this, let's use `strip_prefix`:
```suggestion
        if let Some(s) = ident.0.strip_prefix('$') {
```